.env.example

```
@@ -0,0 +1,11 @@
+# Copy this file to .env and customize values
+# These variables are used by docker-compose to configure the app and database
+
+# Postgres settings
+POSTGRES_USER=Michael
+POSTGRES_PASSWORD=Mickyb22*
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)